src/pages/our-work.js

```diff
@@ -0,0 +1,188 @@
+import React, { Fragment } from 'react'
+import Helmet from 'react-helmet'
+import { StaticQuery, graphql } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+
+import Layout from '../components/layout'
+import GreyBackground from '../components/GreyBackgroundWithoutOffset'
+import { Grid, Row, Col } from '../components/grid'
+import { H1, H3 } from '../components/Typography'
+import Hr from '../components/Common/Hr'
+import CaseStudy from '../components/OurWork/CaseStudy'
+
+const SubHeading = H3.withComponent('h2')
+
+const formatCaseStudies = caseStudies =>
+  caseStudies.edges.map(caseStudyObject => {
+    const caseStudy = caseStudyObject.node
+    return {
+      ...caseStudy,
+      services: caseStudy.services
+        .filter(service => service.title)
+        .map(service => service.title)
+    }
+  })
+
+const OurWork = ({ data }) => {
+  const { site, allContentfulCaseStudy, allContentfulGenericCaseStudy } = data
+
+  const engineeringCaseStudies = formatCaseStudies(allContentfulCaseStudy)
+  const designCaseStudies = formatCaseStudies(allContentfulGenericCaseStudy)
+  const caseStudies = engineeringCaseStudies.concat(designCaseStudies)
+
+  const page = allContentfulCaseStudy.edges[0].node
+
+  return (
+    <Layout backgroundColor="grey">
+      <Helmet
+        title={`${site.siteMetadata.title}  ${
+          page.title ? '- ' + page.title : ''
+        } ${page.seoTitle ? '- ' + page.seoTitle : ''} `}
+        meta={[
+          {
+            name: 'description',
+            content: page.seoMetaDescription
+          }
+        ]}
+      >
+        <html lang="en" />
+      </Helmet>
+      <GreyBackground>
+        <Grid>
+          <Row>
+            <Col width={[1, 1, 1, 1, 11 / 12, 9 / 12, 9 / 12]}>
+              <Padding
+                top={{
+                  smallPhone: 3.5,
+                  smallTablet: 3.5,
+                  tablet: 4,
+                  desktop: 4
+                }}
+                bottom={{
+                  smallPhone: 3.5,
```

Comment:
I think we jus need the `smallPhone` and `tablet` keys, right?, similar for the line 56 (only need smallPhone and tablet)

Reply:
Technically, yes. I guess I just thought being completely declarative would make it easier to discern what was happening.

Declaring only 2 of the breakpoints just means that we're assuming that any newcomer to this project is sufficiently familiar with the `styled-components-spacing` library, to know how this would behave.

I guess I just think it would be useful to [document our conventions somewhere](https://trello.com/c/DrXHIMWe/207-document-how-our-grid-works-should-be-used-in-combination-with-the-other-style-libraries-we-use), especially if we're going to write code in such a way that we're assuming knowledge of 3rd party libraries.

But I'm more than happy to remove these for the sake of consistency and keeping it DRY, because that is arguably more important?

Reply:
> I guess I just think it would be useful to document our conventions somewhere, especially if we're going to write code in such a way that we're assuming knowledge of 3rd party libraries.

^ Completely agree!

I understand the intention, and I'm all up for easing the work on newcomers :) but we also have the other side of the coin - it can be more overwhelming to see more lines of code. I think this has a direct relation with `min-width` media queries so it should not be a hard concept to grasp — but the idea for documenting this should exist in parallel. Do you think the Readme.io is a good place for this ? Or is there another place for docs used at yld?

Reply:
Readme.io is a platform that we (YLD) use to document things about the organisation. I'm not sure that it's necessarily the appropriate place for documentation about the creation of this website.

Or do you mean the `README.md` file within this codebase? Yeah, I think that's probably the best place right now :+1:

Reply:
I'll add something about this to the README.md (if you agree that this is the right place) & then remove these lines?

Reply:
Yeah, the Readme is the most direct and accessible place to document this! go ahead!